src/TBuffer.cpp

```diff
@@ -4149,3 +4182,42 @@ QString TBuffer::bufferToHtml( QPoint P1, QPoint P2, bool allowedTimestamps, int
 
     return s;
 }
+
+const QList<QString> TBuffer::getComputerEncodingNames() {
```

Comment:
Fixed. I made the friendly one first which is why this one came out like that - thanks for spotting an improvement!